### DISCUSSION TITLE
PCB editor theme

### DISCUSSION THREAD

**Discussion by adamalfath** [FiercestT/EasyEdaThemes#11] [General]
Is it possible that the theme applied to the PCB editor as well? I notice that the background color is defined in Canvas Properties side menu and the layer color is on the Layer Manager, is it different than just regular CSS styling?

**Comment by FiercestT** [comment]
It is possible to apply the theme to the PCB editor (and I tested it), but ultimately decided to scrap it at the time. So it is currently not a feature. 

It is not as simple as applying CSS but rather I believe some api calls have to be made, and the layer colors or set on a per project basis. I may be wrong this is just what I recall from when I initially developed the extension.

I didn't think there would be a lot of issue with that as 1) PCB editor is already dark and 2) The default theme is very standard in the PCB design world and it is functionally good. Additionally, I would not want to force users to use a custom PCB theme, but would have to add an option menu for that.

**Reply by adamalfath** [reply]
Ok got it. It's just a matter of personal preferences. Besides, I think I saw that EasyEDA pro version will be implementing the theme globally on schematic & board editor, don't know if the free version will get the same update. Anyways, thanks for this version, really useful for me.
